src/Attachment/index.ts

```diff
@@ -36,6 +37,8 @@ import {
 import { merge, isEmpty, assign, set } from 'lodash'
 import { DEFAULT_BREAKPOINTS } from './decorator'
 
+export const kUploadFolder = 'image_upload_tmp'
```

Comment:
Please I would prefer naming the variable `tempUploadFolder` instead of `kUploadFolder`. Thanks.